scripts/tests.sh

```diff
@@ -57,12 +104,13 @@ function printError() { echo "$(tput setaf 1)$1$(tput sgr0)"; }
 function printTask() { echo "$(tput bold)$(tput setaf 2)$1$(tput sgr0)"; }
 
 printTask "Running commandline tests..."
-"$REPO_ROOT/test/cmdlineTests.sh" &
-CMDLINE_PID=$!
 # Only run in parallel if this is run on CI infrastructure
-if [ -z "$CI" ]
+if [[ -n "$CI" ]]
```

Comment:
Can you explain this change?

Reply:
@chriseth `[` is resolved to `/usr/bin/[` which typically is a symlink to `/usr/bin/test`, hence, every if condition using that construct will spawn a separate process whereas `[[` is a builtin token of bash that is equivalent to `/usr/bin/test` but spares you from consuming process IDs as well as needless process-spawning cycles. (long story short: micro optimization).